EasyClangComplete.py

```diff
@@ -78,6 +78,21 @@ def run(self, edit):
             log.debug(" Nothing to clean")
 
 
+class ApplyFixitCommand(sublime_plugin.TextCommand):
+    """Command to apply fixit at cursor position."""
+
+    def run(self, edit):
+        """All the actual processing is done in on_text_command."""
+
+
+class ReplaceCharactersCommand(sublime_plugin.TextCommand):
+    """Command to actually replace region with fixit value."""
+
+    def run(self, edit, region, value):
+        """Replace given region with given string."""
+        self.view.replace(edit, sublime.Region(*region), value)
```

Comment:
Ok, you're right again. I guess I need some sleep. Sorry for misunderstanding. I will look at it afresh probably tomorrow.